packages/backend/src/apps/formatter/actions/date-time/common/date-time-format.ts

```diff
@@ -46,6 +46,16 @@ const formatConverters = {
         return dateTime
       }
 
+      /**
+       * NOTE: special handling for MyInfo Child, which sends birth date in dd/mm/yyyy format.
+       * Keep this to a single converter for FormSG date fields to minimise confusion for users.
+       */
+      const dtMyInfoChild = DateTime.fromFormat(input, 'dd/MM/yyyy')
+
+      if (dtMyInfoChild.isValid) {
+        return dtMyInfoChild
+      }
+
```

Comment:
should this be a new format instead rather than being "hidden" inside cus now there's no description to tell users that this works for myinfo child date field